### PR TITLE
Drop docs

### PR DESCRIPTION
**Pull request by NicolasMahe**
### Description

Add first version of drop docs

### CODE REVIEW

**Comment by antho1404** [comment]
One small feedback in term of structure I would love to have a similar structure to the rest of the documentation. Other than that, that's great

> Drop
>   - Introduction
>  - Smart contract requirements

**Comment by NicolasMahe** [comment]
updated

File: docs/pages/dashboard/drop.mdx [hunk 1]
```diff
@@ -59,15 +63,15 @@ If not set, the Liteflow Infrastructure will use the number of tokens created be
 
 </Callout>
 
-## Minted by wallet function
+### Minted by wallet function
 
 This function must return the number of tokens minted by a specific wallet during this drop.
 
 The function must be a `view`, have one `input` type `address`, and must return a unique output of type [`integer`](#integer-type).
```
Comment (NicolasMahe) [inline docs/pages/dashboard/drop.mdx]:
```suggestion
The function must be a `view`, have one `input` type `address`, and must return a unique output of type `int` or `uint`.
```

File: docs/pages/dashboard/drop.mdx [hunk 2]
```diff
@@ -32,20 +36,20 @@ function mintToMsgSender(uint8 numberToMint) public payable
...
 necessary.
 
 </Callout>
 
-## Minted function
+### Minted function
 
 This function must return the number of tokens minted by this drop.
 
 The function must be a `view`, have no `inputs`, and must return a unique output of type [`integer`](#integer-type).
```
Comment (NicolasMahe) [inline docs/pages/dashboard/drop.mdx]:
```suggestion
The function must be a `view`, have no `inputs`, and must return a unique output of type `int` or `uint`.
```

**Review by EmmanuelDrouin — APPROVED** [review]

**Comment by EmmanuelDrouin** [comment]
@antho1404 I'm good with the new orga. One link is not working tho "Configure plugins".

Depending on where you see those "how to" going, I would recommend to maybe group those into topics like "collection", "payment method", etc. and have inside those group the related pages.

I'm saying that as the new pages you added are "import a collection", "add a currency", etc. which are starting points but ideally we would have for example:
- Collection
  - Import a collection
  - Edit a collection
  - Remove a collection 
  
 If you follow where I'm going ;)

**Pull request merged**